supertokens_python/recipe/thirdpartypasswordless/types.py

```diff
@@ -24,3 +26,16 @@ def __init__(self, user_id: str,
         self.phone_number: Union[str, None] = phone_number
         self.time_joined: int = time_joined
         self.third_party_info: Union[ThirdPartyInfo, None] = third_party_info
+
+
+_T = TypeVar('_T')
+
+
+class TypeThirdPartyPasswordlessEmailDeliveryInput:
+    pass
```

Comment:
Why is this empty?

Reply:
Updated. Now, we have:
`TypeThirdPartyPasswordlessEmailDeliveryInput = Union[TypeThirdPartyEmailDeliveryInput, TypePasswordlessEmailDeliveryInput]
`